ansible/roles/stack-sunbird/defaults/main.yml

```diff
@@ -296,7 +296,7 @@ dhiti_liveness_readiness:
   livenessProbe:
     httpGet:
       path: /ping
-      port: 4700
+      port: 4201
```

Comment:
change the port number, as it's moved to 3000

Reply:
changed